R/readDaioglou.R

```diff
@@ -0,0 +1,67 @@
+#' Household data from Daioglou et al. 2012
+#'
+#' Global data on population, household size, floor space per capita (m2/cap),
+#' population density (cap/km2), GINI and household expenditures (PPP2005/cap)
+#' from various sources. Quintile 0 stands for the entire population.
+#'
+#' @author Robin Krekeler, Antoine Levesque
+#'
+#' @reference https://doi.org/10.1016/j.energy.2011.10.044
+#'
+#' @param subtype data type
+#'
+#' @importFrom utils read.csv
+#' @importFrom dplyr rename select mutate group_by summarise across %>%
+#' @importFrom rlang .data
+#' @importFrom tidyr replace_na
+#' @importFrom quitte as.quitte
+#' @importFrom magclass as.magpie collapseDim
+#' @export
+
+readDaioglou <- function(subtype = "specific floor space") {
+
+  asNum <- function(string) {
+    as.numeric(gsub(",", "", string))
+  }
+
+  data <- read.csv("Household_Characteristics3.csv",
+                   stringsAsFactors = FALSE,
+                   row.names = NULL) %>%
+    select("region" = "Country",
```

Comment:
In my opinion this is easier to read without the optional quotation marks. This also applies to the following lines and in general.
```suggestion
    select(region = "Country",
```